### PR TITLE
[Fixes 5420] Update component example in documentation

### LINKED ISSUE #5420: Mismatching System Entity Directions
## Expected Behavior
Examples on https://backstage.io/docs/features/software-catalog/descriptor-format should match throughout the page.

## Current Behavior
Currently the entity component example towards the top of the page has `metadata.labels.system` set https://github.com/backstage/backstage/blame/master/docs/features/software-catalog/descriptor-format.md#L47 but further down on the page it's `spec.system` https://github.com/backstage/backstage/blame/master/docs/features/software-catalog/descriptor-format.md#L423. Which is the correct usage? 

## Possible Solution
Update the document to use the same example

## Steps to Reproduce

1. Step 1
2. Step 2
3. ...

## Context
Was following this documentation to prototype adding this functionality into our Backstage instance but ran across the discrepancy.

## Your Environment

- NodeJS Version (v12):
- Operating System and Version (e.g. Ubuntu 14.04):
- Browser Information:

### PR DESCRIPTION
**Pull request by jrusso1020**
## Hey, I just made a Pull Request!
Currently the entity component example towards the top of the page
has metadata.labels.system set
https://github.com/backstage/backstage/blame/master/docs/features/software-catalog/descriptor-format.md#L47
but further down on the page it's spec.system
https://github.com/backstage/backstage/blame/master/docs/features/software-catalog/descriptor-format.md#L423.

This updates them to both use the correct spec.system key/value,
as well as add an example label to get a sense for what labels
look like as well

Signed-off-by: jrusso1020 <jrusso@brex.com>

Fixes #5420 

Screen Shot 2021-04-22 at 9 29 43 AM

Screen Shot 2021-04-22 at 9 29 49 AM

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by adamdmharvey — APPROVED** [review]
As the fix to the reported issue, you nailed it - thanks!

Potentially more broadly, Backstage doesn't even USE labels today, does it?  I see the code is in there to process them, but this description is one of the FIRST thing new users to the system see. I feel like we should just avoid the labels section in metadata entirely so as not to confuse users because as yet I don't believe anything elevates this data to the UI.  So KISS principle and if we get new features later that CAN use it, they can add it, and the model can still say it exists, but no need to make it such a prominent thing - thoughts from the group?

**Comment by jrusso1020** [comment]
@adamdmharvey I'm in agreement, if it's not used lets remove labels for now. Will wait to hear from some of the maintainers though before removing

**Comment by jrusso1020** [comment]
@freben @Fox32 let me know your thoughts when you get a chance! Addressing https://github.com/backstage/backstage/issues/5420

File: docs/features/software-catalog/descriptor-format.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ metadata:
   name: artist-web
   description: The place to be, for great artists
   labels:
-    system: public-websites
+    backstage.io/custom: ValueStuff
```
Comment (Rugvip) [inline docs/features/software-catalog/descriptor-format.md:47]: Let's use `example.com/custom` to make it more clear that you can customize that part of the label as well

**Review by Rugvip — COMMENTED** [review]
Thanks! 👍 

Think it looks good, just a minor nit

File: docs/features/software-catalog/descriptor-format.md (hunk 1 above)
Comment (jrusso1020) [reply docs/features/software-catalog/descriptor-format.md]: @Rugvip addressed! thanks for the feedback

File: docs/features/software-catalog/descriptor-format.md [hunk 2]
```diff
@@ -76,7 +77,7 @@ This is the same entity as returned in JSON from the software catalog API:
     "etag": "ZjU2MWRkZWUtMmMxZS00YTZiLWFmMWMtOTE1NGNiZDdlYzNk",
     "generation": 1,
     "labels": {
-      "system": "public-websites"
+      "backstage.io/custom": "ValueStuff"
```
Comment (Rugvip) [inline docs/features/software-catalog/descriptor-format.md:80]: Needs an update too to match what's above 😅

Comment (jrusso1020) [reply docs/features/software-catalog/descriptor-format.md]: Sorry about that, updated now as well

**Review by Rugvip — APPROVED** [review]
Thanks! 👍

**Pull request merged**